depobs/website/models.py

```diff
@@ -288,6 +302,12 @@ def get_vulnerability_counts(package: str, version: str):
     ).filter(Advisory.package_name==PackageVersion.name
     ).group_by(Advisory.package_name, PackageVersion.version, Advisory.severity)
 
+def get_vulnerabilities(package: str, version: str):
+    return db_session.query(Advisory.package_name, PackageVersion.version, Advisory.severity, Advisory.url, Advisory.title
+    ).filter_by(package_name=package
+    ).filter(PackageVersion.version==version
```

Comment:
`filter_by` takes multiple kwargs, so we might be able to stuff the version filter in the line above. That might be ambiguous though if `PackageVersion` and `Advisory` have a version field.